packages/@aws-cdk/aws-cognito/test/user-pool-client.test.ts

```diff
@@ -42,6 +42,37 @@ describe('User Pool Client', () => {
     expect(() => client2.userPoolClientName).toThrow(/available only if specified on the UserPoolClient during initialization/);
   });
 
+  // eslint-disable-next-line jest/no-focused-tests
+  test.only('client with secret', ()=>{
+    // GIVEN
+    const stack = new Stack();
+    const pool = new UserPool(stack, 'Pool');
+
+    // WHEN
+    const clientWithSecret = new UserPoolClient(stack, 'clientWithSecret', {
```

Comment:
Each of these should be their own test instead of three cases within one test. Please separate them out.

Reply:
Gotcha. Fixed.